Domains.md

```diff
@@ -0,0 +1,52 @@
+# Community domains
+
+This document sets out rules for community domains and the responsibilities and expectations of anyone that maintains and manages the domain settings of official domains used by the Bitcoin Design community. Community domains are used to promote and host content that the community has created and committed to the github.com/bitcoindesign repositories. This currently applies to:
+
+- bitcoin.design
+- bitcoindesigners.org
+
+## The community
```

Comment:
I am not sure if `Community` is proper term here, though it sounds good, but is it fair to communicate it like this? Perhaps `core contributors` or `maintainers` is more appropriate?

In reality community is not the one making decisions in FOSS. Though we were though that FOSS is participatory and that everyone can participate, not everyone has the influence to make changes. The ones in charge of the decision making are always core-contributors, people who in most cases are responsible for 95% of the code and have way better insight on what's going on.

In my personal view, decisions like this cannot be decided by community nor core contributors represent community.

In reality open-source isn't what we were told to. It's not lalaland where anybody vote counts. Votes that count in the decision making process are determined by meritocracy and that's perfectly fine. Some decisions can be public and read-only but as community grows it'll be hard to make "community" consensus.

Reply:
By the way I don't consider my comment as a blocker to this PR. Just a minor food for thoughts as it may become a blocker in the future.

It's a solid document with great outline. ACK.